eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration.php

```diff
@@ -568,4 +569,50 @@ function ($v) {
             ->end()
         ;
     }
+
+    /**
+     * Define Url Alias Slug converter Semantic Configuration.
+     *
+     * The configuration is available at:
+     * <code>
+     * ezpublish:
+     *     url_alias:
+     *         slug_converter_config:
+     *             transformation: name_of_transformation_group_to_use
+     *             separator:  name_of_separator_to_use
+     *             transformation_groups:
+     *                 transformation_group_name: name of existing or new transformation group
+     *                     commands : [] array of commands which will be added to group
+     *                     cleanup_method: name_of_cleanup_method
+     * </code>
+     * @param \Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition $rootNode
+     *
+     * @return null|\Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition|\Symfony\Component\Config\Definition\Builder\NodeBuilder|\Symfony\Component\Config\Definition\Builder\NodeDefinition|\Symfony\Component\Config\Definition\Builder\NodeParentInterface|\Symfony\Component\Config\Definition\Builder\ParentNodeDefinitionInterface|\Symfony\Component\Config\Definition\Builder\VariableNodeDefinition
```

Comment:
Nitpick: returned type is \Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition`